src/pybind/mgr/dashboard/frontend/src/app/shared/api/nvmeof.service.spec.ts

```diff
@@ -27,56 +28,156 @@ describe('NvmeofService', () => {
     expect(service).toBeTruthy();
   });
 
-  // gateways
-  it('should call listGatewayGroups', () => {
-    service.listGatewayGroups().subscribe();
-    const req = httpTesting.expectOne('api/nvmeof/gateway/group');
-    expect(req.request.method).toBe('GET');
-  });
+  describe('gateway', () => {
```

Comment:
```suggestion
  describe(' test gateway APIs', () => {
```
and similarly for others